alembic/versions/9d1483e638af_add_foreign_keys_to_.py

```diff
@@ -0,0 +1,74 @@
+"""Add foreign keys to SampleLaneSequencingMetrics
+
+Revision ID: 9d1483e638af
+Revises: f5e0db62a5a7
+Create Date: 2023-06-01 13:40:25.835471
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = "9d1483e638af"
+down_revision = "f5e0db62a5a7"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Add foreign key constraints to SampleLaneSequencingMetrics table
+    op.add_column(
+        "sample_lane_sequencing_metrics",
+        sa.Column("flow_cell_name", sa.String(length=128), nullable=False),
```

Comment:
This column already exists, not sure if we just need to change it to a foreign key